README.md

```diff
@@ -76,6 +76,34 @@ example below will be announced as `Homepage`.
 <h1 aria-label="Homepage">Project Title</h1>
 ```
 
+## Focus
+
+On page navigation, the plugin will reset focus to the body, imitating default browser behavior.
+When clicking anchor links on the same page, the plugin will focus the target of the link.
+
+> [!TIP]
```

Comment:
Great that we are beginning to use the official `[!TIP]`/`[!CAUTION]`... annotations. We'll need to handle these gracefully in the docs, haven't done that yet if I recall correctly.

Reply:
True, I'll check how this is currently rendered in the docs. If it's bad 🚨 we can open an issue over there.